docs/sources/panels/configure-thresholds/_index.md

```diff
@@ -0,0 +1,96 @@
++++
+title = "Configure thresholds in a Grafana dasboard"
+aliases = ["/docs/grafana/latest/panels/thresholds/", "/docs/grafana/latest/panels/", "/docs/grafana/latest/panels/specify-thresholds/about-thresholds/", "/docs/grafana/latest/panels/specify-thresholds/add-a-threshold/", "/docs/grafana/latest/panels/specify-thresholds/add-threshold-to-graph/", "/docs/grafana/latest/panels/specify-thresholds/delete-a-threshold/"]
+weight = 300
++++
+
+# Configure thresholds in a dashboard
+
+This section includes information about using thresholds in your visualizations.
+
```

Comment:
@jdbaldry - example of where we'd like to insert a TOC for the page. Not a blocker for this PR, just alerting you to the example.

Reply:
@GrafanaWriter Every page get's an automatic TOC in the right side pane:
![Screenshot from 2022-04-22 13-47-47](https://user-images.githubusercontent.com/10999/164708711-0863c95f-6896-48af-a2a4-aed81222acce.png)